tests/common/helpers/constants.py

```diff
@@ -1,6 +1,6 @@
 
 DEFAULT_ASIC_ID = None
-DEFAULT_NAMESPACE = None
+DEFAULT_NAMESPACE = ''
```

Comment:
@arlakshm : FYI . This change is needed. Also it will be consistent with sonic-buildimage